software/speller/data_collection_platform/dcp/api.py

```diff
@@ -70,6 +71,7 @@ def openbci_stop(process_id: int):
     # We need some kind of process manager, and call
     # mp.Process.terminate() and mp.Process.kill() directly.
     # This would also allow us to use mp.Process.is_alive().
+    shared.bci_running[process_id] = False
```

Comment:
We might want to clean up the shared dictionary `bci_running` here as well, e.g. delete everything associated with this process and make sure that the q is empty.

Reply:
I agree, I think this should be implemented with the feature that writes to the database. I don't want to delete the queue without writing it.

Reply:
yea sounds good